relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -378,10 +391,14 @@ pub fn extract_tags(
                     {
                         span_tags.insert(
                             SpanTagKey::RawDomain,
-                            format!("{url_scheme}://{lowercase_host}"),
+                            format!("{url_scheme}://{lowercase_address}"),
                         );
                     }
-                    normalize_domain(domain, port)
+
+                    Some(String::from(concatenate_host_and_port(
+                        Some(domain.to_string().as_str()),
```

Comment:
```suggestion
                        Some(domain.as_ref()),
```

If you just need a `&str` from a `Cow` you don't have to own it first. If you want an owned version of a `Cow` and don't need it later anymore, you should use `into_owned` instead, which does not allocate if you already had a `Cow::Owned`.